### ISSUE TITLE
Paste files doesn't work when multiple items are currently selected

### ISSUE THREAD

**Issue by mcclurgm** [elementary/files#1827]
## What Happened

When I try to paste files into a folder, but multiple items in that folder are selected (have the accent color highlight), it doesn't paste and just sends an error sound. This seems to only happen if 2 or more files are selected. If only one file is selected in the destination, the paste proceeds as usual. However, if I select another, it fails. This appears to be independent of the number of copied files.

## Expected Behavior

Paste should still occur.

## Steps to Reproduce

1. Copy a file from any directory
2. Open a new directory
3. Select at least 2 files in that directory (using shift-click, drag-select, etc.)
4. Try to paste the copied files using the ctrl+v shortcut
5. Error beep sound will play, and the files will not paste.
6. Deselect files in the destination directory and re-select only 1 file
7. Try to paste the copied files using ctrl+v again
8. The paste will proceed

### Logs

I imagine this is it:
```
** (io.elementary.files:190997): WARNING **: 09:04:47.747: AbstractDirectoryView.vala:3054: Cannot paste into a multiple selection
```

## Platform Information

Odin 6.0, up to date as of 9/2/2021.

**Comment by jeremypw** [comment]
This is the expected behaviour. If there are selected items then the action context refers to those selected items and you cannot paste into multiple items. You will see that the context menu does not even show the "Paste into" item. The shortcut activates the same "paste-into" action so likewise fails.  It may be difficult (and potentially confusing) to arrange for different behaviours depending on whether the action is invoked from an accelerator or from a context menu.

**Comment by jeremypw** [comment]
Note that if were to press e.g. `<Ctrl>C` you would expect it to copy the selected items, not the parent folder.

**Comment by mcclurgm** [comment]
Fair enough. I think this is unintuitive, but if it's designed this way, I'll accept it.

I disagree with ctrl+v using "paste into," but that may be a discussion for somewhere else (let me know if so, or if this is something that I shouldn't submit at all). It feels odd that pasting acts one level down the folder hierarchy from where I am, since I always expect the clipboard to act on the current "document" (view). Also, we don't follow this model consistently, since if pasting is supposed to modify the current selection then it should overwrite a currently selected single file (or error) instead of pasting into the parent folder. This makes selection of folders a singular odd case.

To be fair, this specific implementation may be a habit that's just gained from other file managers, since I can't immediately think of any other cases of copy/pasting in a hierarchical setting to draw from.

**Comment by jeremypw** [comment]
If the majority of other file managers act differently to Files then it may be worth getting Design Team opinion on it.

**Comment by jeremypw** [comment]
So if you wanted to paste into a single folder in the view using a shortcut, what would you expect to do (given that `<Ctrl>V` should paste into the parent folder)?

**Comment by mcclurgm** [comment]
To paste into a folder that's shown in the current view, I would enter that folder and use the `<Ctrl>V` shortcut. AFAIK that's how other file managers I've encountered work, but I could just have limited experience/blinders. :shrug:

**Comment by jeremypw** [comment]
I have pushed a PR implementing the requested behaviour (and keeping the existing behaviour with a separate menu option and shortcut).  Lets see if the design team agree.

**Comment by mcclurgm** [comment]
Alright. Thanks for your work implementing that!
